app/containers/Applications/ApplicationWizardConfirmationCardItem.tsx

```diff
@@ -1,125 +1,83 @@
-        {prettyTitle} <span style={{float: 'right'}}>{open ? '+' : '-'}</span>
+    <Card style={{width: '100%', marginBottom: '10px'}}>
+      <Card.Header onClick={() => setIsOpen(!isOpen)}>
+        <Row>
+          <Col md={9}>
+            <h2>{formJson.schema.title}</h2>
+          </Col>
+          <Col md={3} style={{textAlign: 'right'}}>
+            <Button>{isOpen ? 'Expand' : 'Collapse'}</Button>
+          </Col>
+        </Row>
       </Card.Header>
-      <Collapse in={!open}>
+      <Collapse in={!isOpen}>
         <Card.Body>
-          {nested.length > 0 ? (
-            nested.map(nest => (
-              <Table key={nest}>
-                <thead
-                  className="text-center"
-                  style={{
-                    width: '100%',
-                    textTransform: 'capitalize',
-                    textDecoration: 'underline'
-                  }}
-                >
-                  <tr>
-                    <td>
-                      <strong>{nest}</strong>
-                    </td>
-                  </tr>
-                </thead>
-                <tbody>
-                  <tr />
-
-                  {Object.keys(
-                    resultObject[formTitle][formSubtitle][0][nest][0]
-                  ).map(input => (
-                    <tr
-                      key={`${formTitle}, ${formSubtitle}, ${nest}, ${input}`}
-                    >
-                      {renderInputs(formTitle, formSubtitle, input, nest)}
-                    </tr>
-                  ))}
-                </tbody>
-              </Table>
-            ))
-          ) : (
-            <Table>
-              {resultObject[formTitle][formSubtitle].map((item, index) => (
-                <tbody key={`${formTitle} ${formSubtitle} ${item}`}>
-                  <tr />
-                  {inputs.map(input => (
-                    <tr key={`${formTitle} ${formSubtitle} ${input}`}>
-                      {renderInputs(
-                        formTitle,
-                        formSubtitle,
-                        input,
-                        null,
-                        index
-                      )}
-                    </tr>
-                  ))}
-                </tbody>
-              ))}
-            </Table>
-          )}
+          {/*
+      //@ts-ignore JsonSchemaForm typedef is missing customFormats prop */}
+          <JsonSchemaForm
+            omitExtraData
+            liveOmit
+            ArrayFieldTemplate={SummaryFormArrayFieldTemplate}
+            FieldTemplate={SummaryFormFieldTemplate}
+            FormContext={{query}}
+            showErrorList={false}
+            fields={CUSTOM_FIELDS}
+            customFormats={customFormats}
+            schema={schema}
+            uiSchema={uiSchema}
+            ObjectFieldTemplate={FormObjectFieldTemplate}
+            formData={query}
+          >
+            <div style={{display: 'none'}}>
+              <Button type="submit">Submit</Button>
```

Comment:
is the button needed? Can't you just put an empty `span` or fragment?

Reply:
just a display none div is enough apparently. I thought we'd need the button to override the default button, but nope!